incidents/load.py

```diff
@@ -0,0 +1,62 @@
+"""Load functions to load incident data into the database"""
+
+from psycopg2 import connect
+from psycopg2.extensions import connection
+from psycopg2.extras import RealDictCursor
+
+
+def get_db_connection(config: dict[str, str]) -> connection:
+    """Returns a connection to the database."""
```

Comment:
You may remove full stop at the end of the doc string for consistency purposes since you havent used full stops for the others